src/app/api/graphql/route.tsx

```diff
@@ -1,24 +1,24 @@
-import { ApolloServerPluginLandingPageLocalDefault, ApolloServerPluginLandingPageProductionDefault } from '@apollo/server/plugin/landingPage/default'
-
```

Comment:
order of import:- 
import { ApolloServerPluginLandingPageLocalDefault, ApolloServerPluginLandingPageProductionDefault } from '@apollo/server/plugin/landingPage/default'

import { ApolloServer } from '@apollo/server'
import { MongoClient } from 'mongodb'
import { gql } from 'graphql-tag'
import { startServerAndCreateNextHandler } from '@as-integrations/next'